scheduling/job_scheduling.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+"""
+Author : Mohit Kumar
+Job Sequencing Problem implemented in python
+"""
+from collections import namedtuple
+
+
+class Scheduling:
+    def __init__(self, jobs):
+        self.jobs = jobs
+
+    def schedule(self, n, deadline):
+        self.sdl = deadline
+        self.J = [self.jobs[1]]
```

Comment:
```suggestion
        self.j = [self.jobs[1]]
```
In Python, uppercase variable names are reserved for constants (do not change during the runtime of the app).